Source/WebKit/NetworkProcess/cocoa/NetworkDataTaskCocoa.mm

```diff
@@ -593,6 +594,19 @@ static float toNSURLSessionTaskPriority(WebCore::ResourceLoadPriority priority)
     if (m_failureScheduled)
         return;
 
+    CheckedPtr session = m_session.get();
+    if (session) {
+        CheckedPtr storageSession = session->networkStorageSession();
+        if (storageSession && storageSession->cookiesVersion() < m_requiredCookiesVersion) {
+            RELEASE_LOG(Loading, "NetworkDataTaskCocoa::resume is delayed because cookies version is too low");
+            storageSession->setCookiesVersionChangeCallback(m_requiredCookiesVersion, [weakThis = ThreadSafeWeakPtr { *this }] {
```

Comment:
This should be called `addCookiesVersionChangeCallback` since it doesn't overwrite.